constants.yml

```diff
@@ -379,6 +379,8 @@ relationshipType:
     uuid: "9a4b3eea-8a9f-11e8-9a94-a6cf71072f73"
   clinicianToPatient:
     uuid: "2fa6fcea-aa58-11e8-98d0-529269fb1459"
+  motherToPatient:
```

Comment:
It's not really "motherToPatient", it's "motherToChild".  The mother may or may not be a patient, and the child may or may not be a patient.